### ISSUE TITLE
WIND load routines skip variables, should probably default to addmaster=True to use improved metadata

### ISSUE THREAD

**Issue by Beforerr** [spedas/pyspedas#754]
When I tried to use wind 3dp instrument, pyspedas will not load into tplot variables...

```
pyspedas.wind.threedp(["2011-08-26T21:26:00.000Z", "2011-08-26T21:27:30.000Z"],datatype='3dp_pm')
```

```
05-Feb-24 16:57:13: Downloading remote index: https://spdf.gsfc.nasa.gov/pub/data/wind/3dp/3dp_pm/2011/
05-Feb-24 16:57:13: No VAR_TYPE or PARAMETER_TYPE attributes defined for variable Epoch, skipping
05-Feb-24 16:57:13: No VAR_TYPE or PARAMETER_TYPE attributes defined for variable TIME, skipping
05-Feb-24 16:57:13: No VAR_TYPE or PARAMETER_TYPE attributes defined for variable SPIN, skipping
05-Feb-24 16:57:13: No VAR_TYPE or PARAMETER_TYPE attributes defined for variable P_DENS, skipping
05-Feb-24 16:57:13: No VAR_TYPE or PARAMETER_TYPE attributes defined for variable P_VELS, skipping
05-Feb-24 16:57:13: No VAR_TYPE or PARAMETER_TYPE attributes defined for variable P_TENS, skipping
05-Feb-24 16:57:13: No VAR_TYPE or PARAMETER_TYPE attributes defined for variable P_TEMP, skipping
05-Feb-24 16:57:13: No VAR_TYPE or PARAMETER_TYPE attributes defined for variable A_DENS, skipping
05-Feb-24 16:57:13: No VAR_TYPE or PARAMETER_TYPE attributes defined for variable A_VELS, skipping
05-Feb-24 16:57:13: No VAR_TYPE or PARAMETER_TYPE attributes defined for variable A_TENS, skipping
05-Feb-24 16:57:13: No VAR_TYPE or PARAMETER_TYPE attributes defined for variable A_TEMP, skipping
05-Feb-24 16:57:13: No VAR_TYPE or PARAMETER_TYPE attributes defined for variable E_RANGE, skipping
05-Feb-24 16:57:13: No VAR_TYPE or PARAMETER_TYPE attributes defined for variable VC, skipping
05-Feb-24 16:57:13: No VAR_TYPE or PARAMETER_TYPE attributes defined for variable GAP, skipping
05-Feb-24 16:57:13: No VAR_TYPE or PARAMETER_TYPE attributes defined for variable VALID, skipping
```

**Comment by jameswilburlewis** [comment]
Try passing addmaster=True to your call.   This uses a master CDF from SPDF (metadata only, overriding what's in the data CDFs) with much better metadata than the data CDFs.  The WIND data CDFs are missing a lot of metadata, so often a master CDF is necessary.  In this case, it seems to get rid of the warnings.    We should probably make that the default for WIND....I'll keep this task open for that purpose.

**Comment by jameswilburlewis** [comment]
Fix released with pyspedas 1.5.5, now available via pypi
